src/views/Home.vue

```diff
@@ -75,6 +72,20 @@
             </v-row>
           </v-card-text>
         </v-card>
+        <v-card>
+          <v-card-title>{{ $t("views_home.mappooltitle") }}</v-card-title>
+          <div class="seasonMaps" >
```

Comment:
this is stylistic, but I really try to avoid using <div>s where ever possible, maybe <ul> might be right here? Semantic tags help SEO.